content/intro-to-storybook/svelte/en/deploy.md

```diff
@@ -0,0 +1,109 @@
+
+<div class="aside">
+<strong>Did you setup Chromatic testing earlier?</strong>
+<br/>
+🎉 Your stories are already deployed! Chromatic securely indexes your stories online and tracks them across branches and commits. Skip this chapter and go to the <a href="/svelte/en/conclusion">conclusion</a>.
+</div>
+
+## Exporting as a static app
+
+To deploy Storybook we first need to export it as a static web app. This functionality is already built into Storybook, we just need to activate it by adding a script to `package.json`.
+
+```javascript
+// package.json
+
+{
+  "scripts": {
+   "build-storybook": "build-storybook -c .storybook -s public -o storybook-static"
+  }
+}
+```
+
+Now when you build Storybook via `npm run build-storybook`, it will output a static Storybook in the `storybook-static` directory.
+
+## Continuous deploy
+
+We want to share the latest version of components whenever we push code. To do this we need to continuous deploy Storybook. We’ll rely on GitHub and Netlify to deploy our static site. We’re using the Netlify free plan.
+
+## GitHub
+
+Before deploying to Netlify we need to take some additional steps. If you're following along from the previous testing chapter jump to setting up a repository on GitHub.
+
+### Initialize Repo
+
+First you want to setup Git for your project in the local directory.
+
+```bash
+$ git init
+```
+
+Next add files to the first commit.
+
+```bash
+$ git add .
+```
+
+Now commit the files.
+
+```bash
+$ git commit -m "taskbox UI"
+```
+
+### Setup a repository in GitHub
+
+Go to GitHub and setup a repository [here](https://github.com/new). Name your repo “taskbox”.
+
+![GitHub setup](/intro-to-storybook/github-create-taskbox.png)
+
+In the new repo setup copy the origin URL of the repo and add it to your git project with this command:
+
+```bash
+$ git remote add origin https://github.com/<your username>/taskbox.git
+```
+
+Finally push the repo to GitHub
+
+```bash
+$ git push -u origin master
+```
+
+### Netlify
+
+Netlify has a continuous deployment service built in which will allow us to deploy Storybook without needing to configure our own CI.
+
+<div class="aside">
+If you use CI at your company, add a deploy script to your config that uploads <code>storybook-static</code> to a static hosting service like S3.
+</div>
+
+[Create an account on Netlify](https://app.netlify.com/start) and click to “create site”.
+
+![Netlify create site](/intro-to-storybook/netlify-create-site.png)
+
+Next click the GitHub button to connect Netlify to GitHub. This allows it to access our remote Taskbox repo.
+
+Now select the taskbox GitHub repo from the list of options.
+
+![Netlify connect to repo](/intro-to-storybook/netlify-account-picker.png)
+
+Configure Netlify by highlighting which build command to run in its CI and which directory the static site is outputted in. For branch choose `master`. Directory is `storybook-static`. Build command use `yarn build-storybook`.
```

Comment:
Hi @jonniebigodes  please change it for Directory is `.storybook-static` as per PR https://github.com/chromaui/learnstorybook.com/pull/222

Reply:
will do it shortly. Forgot about that.

Reply:
Should be fixed now. Marking this as resolved if you don't mind

Reply:
sounds good!